client/state/jetpack/connection/actions.js

```diff
@@ -112,3 +113,20 @@ export const disconnect = siteId => {
 			} );
 	};
 };
+
+/**
+ * Change the jetpack master user.
+ *
+ * @param {number} siteId the site ID
+ * @param {number} newOwnerDotorgId the dotorg user ID of the new owner
+ * @param {string} newOwnerWpcomDisplayName display name for UI messages
+ * @return {Object} action object
+ */
+export const changeOwner = ( siteId, newOwnerDotorgId, newOwnerWpcomDisplayName ) => {
```

Comment:
Since the "owner" is the user-facing term, but the technical term is "master user", using both terms in naming and descriptions could create some confusion to a Calypso developer who's not very familiar with Jetpack. I'm thinking it might make more sense to use the technical term for naming/documenting functional bits. What do you think?

Reply:
Mm good point. However, [The API](https://github.com/Automattic/jetpack/pull/9578) is named _connection owner_ so we can probably use that term consistently in Calypso.

Reply:
Oh well - seems like the discrepancy started from there. Since we have to choose one or the other, I'm leaving this to your best judgement.